styles/itch.io/catppuccin.user.less

```
@@ -0,0 +1,344 @@
+          fade(@accent, 40%)
+        ),
+        url("https://static.itch.io/images/app-banner.png"),
+        linear-gradient(to right, @surface2, @accent);
+
+      .outline_button {
+        border-color: @text;
+        color: @text;
+
+        &:hover {
+          color: @crust;
+          background-color: @accent;
+          border-color: @accent;
+        }
+      }
+    }
+
+    .index_page .index_sidebar .sidebar_browse_group .browse_filters a:hover,
+    .index_page .sidebar_social .get_app:hover,
+    .index_page .sidebar_social .icon:hover,
+    .index_page .index_sidebar .blog_post_section .post_link:hover,
+    .index_page .index_sidebar .blog_post_section p a:hover {
+      color: @accent;
+    }
+
+    .index_game_cell_widget .meta_tag.web_tag,
+    .index_game_cell_widget .meta_tag.demo_tag,
+    .index_game_cell_widget .meta_tag.free_tag {
+      background-color: @crust;
+    }
+
+    .index_game_cell_widget .meta_tag.price_tag {
+      background-color: @crust !important;
+      background-image: none;
+      color: @text;
+    }
+
+    .index_game_cell_widget .sale_flag {
+      border-top-color: @overlay2;
+      border-left-color: @overlay2;
+      background-color: @accent;
+      color: @base;
+    }
+
+    .browse_related_tags_widget .tag_segmented_btn > a:hover {
+      background-color: @accent;
+      color: @base;
+    }
+
+    .browse_base_page .youtube_banner {
+      box-shadow: 5px 5px 0 @accent;
+      background-image: linear-gradient(to right, @blue, fade(#0fff, 60%)) 50%
+        50% no-repeat;
+
+      .new {
+        background-color: @base;
+        color: @text;
+      }
+    }
+
+    .game_grid_widget .game_cell {
+      .game_cell_tools .action_btn {
+        background-color: @base;
+        color: @text;
+        border-color: @accent;
+      }
+
+      .meta_tag {
+        > div {
+          background-image: linear-gradient(
+            to top right,
+            @crust 0%,
+            @crust 100%
+          );
+        }
+        .sale_tag {
+          background-image: linear-gradient(
+            to top right,
+            @accent 0%,
+            @accent 100%
+          );
+          color: @base;
+        }
+      }
+    }
+
+    .browse_filter_group_widget > ul li:not(.active) > a:hover,
+    .browse_filter_group_widget > ul li:not(.active) > button:hover,
+    .browse_filter_group_widget > ul li:not(.active) .filter_append:hover {
+      background-color: @surface0;
+      color: @text;
+    }
+
+    .browse_filter_group_widget > ul li.active > a,
+    .browse_filter_group_widget > ul li.active > button,
+    .browse_filter_group_widget > ul li.active .filter_append {
+      color: @accent;
+    }
+
+    .browse_filter_group_widget .filter_clear>.svgicon {
+      background-color: @text;
+    }
+
+    .browse_filter_group_widget .filter_clear {
+      color: @base;
+    }
+
+    .browse_filter_group_widget.active,
+    .main .inner_column {
+      border-color: fade(@text, 80%);
```

```suggestion
      border-color: @surface2;
```

You should almost never be `fade`-ing a non-accent color when you can instead use a lighter or darker shade. Also we typically use surface colors for borders.

---

got it!